src/main/resources/plugin.yml

```diff
@@ -2,7 +2,6 @@ name: ViaVersionStatus
 main: com.bobcat00.viaversionstatus.ViaVersionStatus
 version: ${project.version}
 author: Bobcat00
-api-version: 1.13
```

Comment:
Restore this for the reason listed earlier.

Reply:
Corrected in commit `651833cb6e1d2420f0f13111cd8999e9aedd7ed1`.